independent-projects/arc/processor/src/main/java/io/quarkus/arc/processor/BeanConfiguratorBase.java

```diff
@@ -48,13 +51,15 @@
     protected boolean forceApplicationClass;
     protected String targetPackageName;
     protected Integer priority;
+    protected final List<TypeAndQualifiers> injectionPoints;
```

Comment:
Shouldn't this be a `Set`? Currently, you can add the same IP twice which doesn't make much sense.

Reply:
You're right. It would make sense if you could distinguish these IPs at runtime, e.g. to inject two `@Dependent` bean instances of the same bean. But we don't support this use case.

Reply:
Done.